packages/distributor/solana/clients/BaseDistributorClient.ts

```diff
@@ -354,11 +327,33 @@ export default class SolanaDistributorClient {
       tokenProgram: tokenProgramId,
     };
 
-    ixs.push(clawback(accounts, this.programId));
+    const clawbackInstruction = await this.prepareClawbackInstruction(accounts);
+
+    ixs.push(clawbackInstruction);
+
+    //ixs.push(clawback(accounts, this.programId));
```

Comment:
leftover comment - will remove in next iteration

Reply:
what is `next iteration`? Are you going to create PRs into this PR or what is the plan?

Reply:
I am expecting to push some fixes during integration testing (or maybe not we'll see) - if not this comment is just here as "I saw it, will fix" :D